### PR TITLE
Adds *funny* meat as a hunting result (for funny meat lovers)

### PR DESCRIPTION
**Pull request by tornikeo**
#### Summary
Content "Cannibal companions sometimes bring funny meat"

#### Purpose of change

Meat is back on the menu boys (and girls)!

Your cannibal companion, when assigned to hunt in a faction camp, will sometimes bring back funny looking meat. 

#### Describe the solution

Make new mon group for hunt results. Add feral humans in it. Modify c++ logic to allow cannibals to sometimes bring back the funny meat.

#### Describe alternatives you've considered

None.

#### Testing

TODO.

#### Additional context

This is a very meaty issue - still working on this...

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -64,4 +64,7 @@
     "github-actions.remote-name": "upstream",
     // NOTE: This disable the plugins formatting so astyle is used.
     "C_Cpp.formatting": "disabled",
+    "files.associations": {
+        "cmath": "cpp"
```
Comment (Maleclypse) [inline .vscode/settings.json:68]: Is this change intended?

Comment (tornikeo) [reply .vscode/settings.json:68]: Ah, when did that slip in? No, def not intended. Will remove once I'm home

Comment (tornikeo) [reply .vscode/settings.json]: Should be good now @Maleclypse

**Comment by kevingranade** [comment]
For this to be reasonably consistent it would need a ton of nuance around for example this exposing the cannibal and reprecussions of that happening up to and including that npc being banished from the community.  That begs the question,  is all that worth adding? No.

**Comment by tornikeo** [comment]
Wh.. why would anyone "expose" or banish a cannibal in the CDDA world? :D cannibalism seems like the most sustainable trait in this world.

But sure, yes, this couls use more nuance, and this was supposed to be a joke PR either way.

**Comment by esotericist** [comment]
@tornikeo do not submit joke PRs.
